README.md

```diff
@@ -46,7 +46,7 @@ ipinfoWrapper.lookupASN("AS7922").then((response: AsnResponse) => {
 ```javascript
 let { IPinfoWrapper } = require("node-ipinfo");
 
-let ipinfo = new IPinfo("MY_TOKEN");
+let ipinfo = new IPinfoWrapper("MY_TOKEN");
```

Comment:
Beautiful, thank you. I think I made the same mistake below as well - could you add a commit for those as well?

Reply:
Sure 👍🏻